### PR TITLE
Update readme and add migrations to codebase

### PR DESCRIPTION
**Pull request by Corb3nik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,29 @@
...
+authentication service.
+
+To use OTA-University on your Slack, you'll need to create a
+Slack Application and fetch its `SLACK_CLIENT_ID` and `SLACK_CLIENT_SECRET`.
+More information [here](https://api.slack.com/applications).
+
+# Deployment
+- `git clone https://github.com/OpenToAllCTF/OTA-University`
+- `cd OTA-University`
+- `pip3 intall -r REQUIREMENTS.txt`
```
Comment (reznok) [inline README.md]: intall -> install

File: README.md [hunk 2]
```diff
@@ -1,33 +1,29 @@
...
+- `cp ota_university/config.yaml.template ota_university/config.yaml`
+- Fill in the settings in ota_university/config.yml
+- `python3 manage.py migrate`
+- `python3 manage.py runserver`
+
+# Creating an Administrator Account
+To create new admin accounts, use the following command :
+`python3 manage.py createsuperuser`
```
Comment (reznok) [inline README.md]: Should we add: "Local admins can login via the /admin endpoint" ?

Though this will be going away pretty quickly

Comment (Corb3nik) [reply README.md]: Sure!

**Comment by reznok** [comment]
fixes #8

**Pull request merged**